### PR TITLE
[Doc] Add return_proba argument explanation

### PR DESCRIPTION
**Pull request by zhjwy9343**
*Issue #, if available:*

*Description of changes:*
This PR fix the documentation of `return_proba`:
- Add the return_proba argument;
- Add a note on eval_metric.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/configuration/configuration-run.rst [hunk 1]
```diff
@@ -307,6 +311,10 @@ Classification and Regression Task
     - Yaml: ``imbalance_class_weights: 0.1,0.2,0.3``
     - Argument: ``--imbalance-class-weights 0.1,0.2,0.3``
     - Default value: ``None``
+- **return_proba**: For classification task, this configuraiton determines whether to return prediction probability or the argmax prediction. Set `true`` to return predictions probability and `false` to return the argmax prediction.
```
Comment (GentleZhu) [inline docs/source/configuration/configuration-run.rst:314]: typo: configuration. Can we say class prediction using argmax instead of argmax prediction?

Comment (zhjwy9343) [reply docs/source/configuration/configuration-run.rst]: I rewrote with the terms extracted from scklearn's documentation.

**Comment by zhjwy9343** [comment]
Also remove the note because we decide the return_proba should have nothing to do with evaluation metrics. We need to set its value automatically based on the metrics. No need users to be aware of this.

File: docs/source/configuration/configuration-run.rst (hunk 1 above)
Comment (GentleZhu) [reply docs/source/configuration/configuration-run.rst]: LGTM. please correct "configuraiton" into "configuration"

**Review by GentleZhu — APPROVED** [review]

File: docs/source/configuration/configuration-run.rst (hunk 1 above)
Comment (zhjwy9343) [reply docs/source/configuration/configuration-run.rst]: fixed. thanks

**Pull request merged**